frontend/services/lua_service_wrapper/Lua_Service_Wrapper.hpp

```diff
@@ -84,8 +80,7 @@ class Lua_Service_Wrapper final : public AbstractFrontendService {
 
     int m_service_recursion_depth = 0;
 
-    // auto-deleted opaque ZMQ networking object from LuaHostService.h
-    std::unique_ptr<void, std::function<void(void*)>> m_network_host_pimpl;
+    std::unique_ptr<megamol::frontend::LuaRemoteConnectionsBroker> m_network_host;
```

Comment:
I guess if we directly use the `LuaRemoteConnectionsBroker` type we also dont need the `std::unique_ptr` anymore?